packages/react-composites/src/composites/CallComposite/Error.tsx

```diff
@@ -36,10 +37,13 @@ export function Error(props: ErrorProps): JSX.Element {
       <Stack tokens={upperStackTokens}>
         <div className={endCallTitleStyle}>{title}</div>
         <Stack horizontal tokens={buttonsStackTokens}>
-          <PrimaryButton className={buttonStyle} onClick={props.rejoinHandler}>
-            <Video20Filled primaryFill="currentColor" className={videoCameraIconStyle} />
-            {rejoinCall}
-          </PrimaryButton>
+          <PrimaryButton
+            className={buttonStyle}
+            styles={buttonWithIconStyles}
+            text={rejoinCall}
```

Comment:
Also moved to locale strings file (maybe worth filing a bug with these instances in them)